### PR TITLE
Fix circular dependancy + Update README

### PR DESCRIPTION
**Pull request by JayaShakthi97**
### Purpose
- Fix circular dependency issue
trace

- Update README callback URL regex

### Related Issues
- None

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- None

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Review by brionmario — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -73,13 +73,13 @@ If you are building [product-is](https://github.com/wso2/product-is), the built
     **My Account**
 
     ```
-    regexp=(https://localhost:9443/myaccount/login|https://localhost:9443/myaccount/logout|https://localhost:9000/myaccount/login|https://localhost:9000/myaccount/logout|https://localhost:9000/myaccount)
+    regexp=(https://localhost:9443/console|https://localhost:9443/t/(.*)/console|https://localhost:9443/console/login|https://localhost:9443/t/(.*)/console/login|https://localhost:9001/console|https://localhost:9001/t/(.*)/console|https://localhost:9001/console/login|https://localhost:9001/t/(.*)/console/login)
```
Comment (Yasara-Y) [inline README.md:76]: This has to be the regex of console, not myaccount.

File: README.md [hunk 2]
```diff
@@ -73,13 +73,13 @@ If you are building [product-is](https://github.com/wso2/product-is), the built
...
     ```
 
     **Console**
 
     ```
-    regexp=(https://localhost:9443/console/login|https://localhost:9443/console/logout|https://localhost:9001/console/login|https://localhost:9001/console/logout|https://localhost:9001/console)
+    regexp=(https://localhost:9443/myaccount|https://localhost:9443/t/(.*)/myaccount|https://localhost:9443/myaccount/login|https://localhost:9443/t/(.*)/myaccount/login|https://localhost:9000/myaccount|https://localhost:9000/t/(.*)/myaccount|https://localhost:9000/myaccount/login|https://localhost:9000/t/(.*)/myaccount/login)
```
Comment (Yasara-Y) [inline README.md:82]: This is the regexp of myaccount.

File: README.md (hunk 1 above)
Comment (JayaShakthi97) [reply README.md:76]: Already with eab38e16c09ec69679233f8056128f8cc1c46c20

File: README.md (hunk 2 above)
Comment (JayaShakthi97) [reply README.md:82]: Already with eab38e16c09ec69679233f8056128f8cc1c46c20

**Pull request merged**